README.md

```diff
@@ -72,15 +79,73 @@ After setting up your environment, you can verify your setup by sending a test m
   - **S3 URL**: Replace the S3 URL in the `--message` argument with the URL of the image file you want to test.
 
 2. Execute the following command, substituting your actual values:
-
     ```bash
-    python3 bin/local_cli.py --topic-arn <YOUR_TOPIC_ARN> --s3-uri <YOUR_S3_URI>
+    python3 bin/intake_cli.py --topic-arn <YOUR_TOPIC_ARN> --s3-uri <YOUR_S3_URI>
     ```
 
-3. **Expected Output**:
+3. Validate Expected Output:
```

Comment:
Is there a quick command to check the expected output without going into the console?

Reply:
We can follow up on additional CLI functionality